src/MPBWrapper.jl

```diff
@@ -194,7 +195,7 @@ MPB.getrawsolver(m::CbcMathProgModel) = m.inner
 
 function MPB.setwarmstart!(m::CbcMathProgModel, v)
     if any(isnan, v)
-        Base.warn_once("Ignoring partial starting solution. Cbc requires a feasible value to be specified for all variables.")
+        Compat.@warn("Ignoring partial starting solution. Cbc requires a feasible value to be specified for all variables.")
```

Comment:
To be identical to `warn_once`, you should use `maxlog=1` if I'm not mistaken

Reply:
True, done.